plugins/plugin-webpack/README.md

```diff
@@ -40,15 +28,7 @@ module.exports = {
 };
 ```
 
-The options object is optional.
-
-### Default Build Script
-
-```json
-{
-  "scripts": {"bundle:*": "@snowpack/plugin-webpack"}
-}
```

Comment:
@FredKSchott I had no idea what this meant or what it was for so I removed it, let me know if it’s important and I’ll add it back

Reply:
Good job!

Reply:
oh wow, this is pretty old. Great catch!